Model/src/GamePackage/Player.java

```diff
@@ -1,4 +1,265 @@
 package GamePackage;
 
+import java.util.ArrayList;
+
+import BoardPackage.BoardSquare;
+import BoardPackage.PropertySquare;
+import CardPackage.Card;
+
+/**********************************************************************
+ * The Player class will keep track of each player's money, properties,
+ * and other stats.
+ * 
+ * @author Dustin Ecker
+ * @version 2/11/2018
+ *********************************************************************/
 public class Player {
+	
+	/** Player's name */
+	private String displayName;
```

Comment:
Should the display name be final?

Reply:
I agree

Reply:
I though we where going to allow the player to choose their own name ?

Reply:
We are, once its chosen and set, it wont change. right?